circuits/ts/__tests__/ProcessMessages.test.ts

```diff
@@ -47,6 +47,32 @@ describe("ProcessMessage circuit", function test() {
     ]
   >;
 
+  let circuitNonQv: WitnessTester<
+    [
+      "inputHash",
+      "packedVals",
+      "pollEndTimestamp",
+      "msgRoot",
+      "msgs",
+      "msgSubrootPathElements",
+      "coordPrivKey",
+      "coordPubKey",
+      "encPubKeys",
+      "currentStateRoot",
+      "currentStateLeaves",
+      "currentStateLeavesPathElements",
+      "currentSbCommitment",
+      "currentSbSalt",
+      "newSbCommitment",
+      "newSbSalt",
+      "currentBallotRoot",
+      "currentBallots",
+      "currentBallotsPathElements",
+      "currentVoteWeights",
+      "currentVoteWeightsPathElements",
+    ]
```

Comment:
I think the type is same as circuit. could we define the type else where?

Reply:
can do that yea